core_impl/src/test/java/com/google/instrumentation/stats/StatsManagerImplTest.java

```diff
@@ -55,14 +53,6 @@ public void testRegisterUnsupportedViewDesciptor() throws Exception {
   }
 
   @Test
-  public void testRegisterDuplicatedViewDesciptor() throws Exception {
-    statsManager.registerView(RpcConstants.RPC_CLIENT_ROUNDTRIP_LATENCY_VIEW);
-
-    thrown.expect(IllegalArgumentException.class);
-    statsManager.registerView(RpcConstants.RPC_CLIENT_ROUNDTRIP_LATENCY_VIEW);
-  }
```

Comment:
I think this test is still useful for checking that the second call doesn't throw an exception.